homeassistant/components/modbus/sensor.py

```diff
@@ -252,16 +251,3 @@ def update(self):
             self._value = f"{val:.{self._precision}f}"
 
         self._available = True
-
-    def _set_unavailable(self):
```

Comment:
Please see comment in climate.py

Reply:
Somehow there is a difference between my version (on my disk) and what you are looking at. Just to be sure, I will do another "git push -f". 

Wait a sec. can you be looking at something else, because you have loaded it as a branch. Be aware I use "git rebase" quite often (as long as a PR is open and not in final review) to keep the commits more logical.